### ISSUE TITLE
fnf cutscene test

### ISSUE THREAD

**Issue by luckydog7** [luckydog7/Funkin-android#771]
apk from this video https://youtu.be/IDdIr2cDzSY
https://www.mediafire.com/file/17as0bnraxuhmrq/Funkin-debug.apk/file

if it has any problems, please write

**Comment by FridayNightVibin** [comment]
This is so cool

**Comment by FridayNightVibin** [comment]
and can u pls fix week 5

**Comment by FridayNightVibin** [comment]
When i complete coaca it crashes

**Comment by FridayNightVibin** [comment]
Wow im really impressed in myself

**Comment by FridayNightVibin** [comment]
Im the first one to see this

**Comment by MrTks216** [comment]
Alright, i gonna download and test it

**Comment by georgechen1236** [comment]
Thank you

**Comment by FridayNightSans** [comment]
DANGIT, RECORDING TIME

**Comment by NoahEggz** [comment]
:happi: :)

**Comment by georgechen1236** [comment]
Yes I'm so happy

**Comment by MrTks216** [comment]
Okay done, work pretty well, no lag. Great job Luckydog

**Comment by FridayNightSans** [comment]
Great job, it kinda normally... but i think you should let it in Friday Night Funkin'/assets/week 7/cutsense/ughCutsense.mp4
I think it shouls be work like that ;-; idk

**Comment by FridayNightVibin** [comment]
It just shows mine this

**Comment by FridayNightVibin** [comment]
![Uploading Screenshot_20210513_224624_com.android.packageinstaller.jpg…]()

**Comment by FridayNightVibin** [comment]
Bruh what do i do

**Comment by MrTks216** [comment]
> Bruh what do i do

Bruh, you have to uninstall another fnf on your phone.

**Comment by sonicyt1234** [comment]
It just crashes for me

**Comment by FridayNightVibin** [comment]
Bruh the original one

**Comment by FridayNightVibin** [comment]
No bruh i will just wait

**Comment by FridayNightVibin** [comment]
I wont risk it

**Comment by FridayNightVibin** [comment]
And i have a tablet not a phone

**Comment by DiosDonutSlave** [comment]
It works well!

**Comment by claus296** [comment]
Doesn't wanna work for me
![Screenshot_20210513-205205](https://user-images.githubusercontent.com/81391522/118204915-4f2fe900-b42d-11eb-87df-5eebb026fbb4.png)

**Comment by DabKnight** [comment]
> Doesn't wanna work for me
> ![Screenshot_20210513-205205](https://user-images.githubusercontent.com/81391522/118204915-4f2fe900-b42d-11eb-87df-5eebb026fbb4.png)

Do it again and again until it works.
Trust me it works because it always happens to me then i just try to do it again.

**Comment by DabKnight** [comment]
![Screenshot_20210514_094355](https://user-images.githubusercontent.com/82740118/118208225-f39b4500-b498-11eb-8f73-a49c970f29a7.jpg)
It doesnt wanna load.

**Comment by TunnelTube** [comment]
There Is An Issue, First Of, The cutscene OR video is slow, which makes the Sound of the video or cutscene, Un-synced. Pls fix that.

**Comment by Ruv023** [comment]
> apk from this video https://youtu.be/IDdIr2cDzSY
> https://www.mediafire.com/file/17as0bnraxuhmrq/Funkin-debug.apk/file
> 
> if it has any problems, please write

So far so good. Looking great :)

**Comment by VannyTheBunny09** [comment]
im so happy

**Comment by VannyTheBunny09** [comment]
> apk from this video https://youtu.be/IDdIr2cDzSY
> https://www.mediafire.com/file/17as0bnraxuhmrq/Funkin-debug.apk/file
> 
> if it has any problems, please write

bruh im gonna explode. im so happy

**Comment by DabKnight** [comment]
> There Is An Issue, First Of, The cutscene OR video is slow, which makes the Sound of the video or cutscene, Un-synced. Pls fix that.

I think its your phone.
Because it works fine :/

**Comment by TunnelTube** [comment]
> > There Is An Issue, First Of, The cutscene OR video is slow, which makes the Sound of the video or cutscene, Un-synced. Pls fix that.
> 
> I think its your phone.
> Because it works fine :/

Excuse Me Sir! My phone has 51gb free Storage And 4gb Ram. What Else Does It Want?

**Comment by DabKnight** [comment]
Oh.
Idk :/

**Comment by VitalieW0WKid** [comment]
So if the official cutscenes are gonna be only video with audio,im gonna make rickroll mod out of it

**Comment by claus296** [comment]
Yes

**Comment by GlitzHere** [comment]
Runs really well on my device!!

**Comment by GlitzHere** [comment]
I played it and I really like how you can see the hitboxes before touching the screen lok

**Comment by FridayNightVibin** [comment]
Wait does it have week 7

**Comment by Kris6528** [comment]
Yeah,like the game assets are in the data folder will he more easier and less ram usage

**Comment by POKO-dev** [comment]
> Yeah,like the game assets are in the data folder will be more easier and less ram usage

Nice its not a hard jobe to add
